examples/tree_1d_dgsem/elixir_shallowwater_parabolic_bowl.jl

```diff
@@ -0,0 +1,109 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.81)
+cfl = 0.3
+
+
+function parabolic_bowl_analytic_1D_H(gravity, x,t)
```

Comment:
Is this function for later comparison of the approximate solution? I'm not sure it needs to live in the elixir in this case as it would be part of postprocessing the solution data.